src/main/java/antycaptcha/utilities/ByConverter.java

```diff
@@ -1,2 +1,31 @@
-package antycaptcha.utilities;public class ByConverter {
+package antycaptcha.utilities;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class ByConverter {
+
+    public static By toByVal(WebElement we) {
+        String[] data = we.toString().split(" -> ")[1].replace("]", "").split(": ");
+        String locator = data[0];
+        String term = data[1];
+
+        switch (locator) {
+            case "xpath":
+                return By.xpath(term);
+            case "css selector":
+                return By.cssSelector(term);
+            case "id":
+                return By.id(term);
+            case "tag name":
+                return By.tagName(term);
+            case "name":
+                return By.name(term);
+            case "link text":
+                return By.linkText(term);
+            case "class name":
+                return By.className(term);
+        }
+        return (By) we;
+    }
 }
```

Comment:
nie bardzo widze sens tej klasy, lepiej sobie zrobic fabryke np:

```
public class ElementWrapper {
    private final By locator;
    private final WebElement element;

    public ElementWrapper(By locator, WebElement element) {
        this.locator = locator;
        this.element = element;
    }

    public By getLocator() {
        return locator;
    }

    public WebElement getElement() {
        return element;
    }
}
```

Reply:
ale nadal musisz mi wyjaśnić po co

Reply:
nie no to wszystko w selenium powinno byc jawnie, to nie do konca jest dobry kierunek